app/routes/goal_routes.py

```diff
@@ -1 +1,100 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, request, abort, make_response
+from ..db import db
+from ..models.goal import Goal
+from ..models.task import Task
+from .route_utilities import validate_model
+
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    try:
+        new_goal = Goal.from_dict(request_body)
+    except KeyError:
+        return jsonify({"details": "Invalid data"}), 400
+
+    db.session.add(new_goal)
+    db.session.commit()
+
+    return jsonify(new_goal.to_dict()), 201
```

Comment:
Wave 7 asks folks to implement the refactors we covered for hello-books and solar-system. I see that the functions exist in the `route_utilities` file, but the functions are not implemented in the routes. 

I would like to see you revisit the project and implement the helper functions inside the routes to D.R.Y. up the create routes for Goals and Tasks to meet the project requirements.